### ISSUE TITLE
Multilang support?

### ISSUE THREAD

**Issue by ViBE-HU** [Elinvention/efiboots#39]
I'm using your fine tool for many years now and I also translated the script for myself but it would be cool if we can "spread the word". Any chances?

**Comment by Elinvention** [comment]
Yes, it could be done by integrating the gettext library. Maybe one day...

**Comment by Elinvention** [comment]
I came back to this project and added gettext yesterday. Now we need translations!
